pkg/boot/syslinux/testdata/debian-10-install/isolinux/addrk.cfg

```diff
@@ -0,0 +1,9 @@
+label expertdark
```

Comment:
Can you add a link to where these scripts came from and whether they have a license?

Reply:
I installed Debian 10 in a VM. These are the syslinux scripts used by the ISO installer. I'm not sure they have a license...